aspnetcore/fundamentals/minimal-apis/7.0-samples/todo-group/Dtos.cs

```diff
@@ -0,0 +1,12 @@
+internal class TodoDto
```

Comment:
```suggestion
namespace Data;

internal class TodoDto
```

These classes should probably also go in their own files. You can put them in a separate DTO folder if we want, but I don't think that's necessary. I'd probably just put it in the `Data` folder with the non-DTO types. The suffix should make it pretty clear what the purpose is.

Reply:
@halter73  I was told about making it portable for users to try out by @Rick-Anderson but I don't know if I should create more files or make it concise.
I need help on this to be clear on what to do next

Reply:
@sammychinedu2ky  put the classes in their own file. On rare occasions when the class is very small and we want a very small sample we mix them, but in general we like to follow best practices which is to put classes in their own file in an appropriate directory.

Reply:
ok